tests/test_sensors.py

```diff
@@ -72,5 +99,92 @@ def test_imu_sensor(show_viewer):
     for _ in range(80):
         scene.step()
 
+    assert_allclose(imu_skewed.read()["lin_acc"], -GRAVITY, tol=1e-7)
     assert_allclose(imu_biased.read()["lin_acc"], torch.tensor([BIAS[0], BIAS[1], BIAS[2] - GRAVITY]), tol=1e-7)
     assert_allclose(imu_biased.read()["ang_vel"], BIAS, tol=1e-5)
+
+    scene.reset()
+
+    assert_array_equal(imu_biased.read()["lin_acc"], 0.0)  # biased, but cache hasn't been updated yet
+    assert_array_equal(imu_delayed.read()["lin_acc"], 0.0)
+    assert_array_equal(imu_noisy.read()["ang_vel"], 0.0)
+
+    scene.step()
+    assert_allclose(imu_biased.read()["lin_acc"], BIAS, tol=1e-7)
+
+
+@pytest.mark.required
+def test_rigid_tactile_sensors_gravity_force(show_viewer):
+    """Test if the sensor will detect the correct forces being applied on a falling box."""
+    GRAVITY = -10.0
+    BIAS = (0.1, 0.2, 0.3)
+    NOISE = 0.01
+
+    scene = gs.Scene(
+        sim_options=gs.options.SimOptions(
+            dt=1e-2,
+            substeps=1,
+            gravity=(0.0, 0.0, GRAVITY),
+        ),
+        profiling_options=gs.options.ProfilingOptions(show_FPS=False),
+        show_viewer=show_viewer,
+    )
+
+    scene.add_entity(morph=gs.morphs.Plane())
+
+    box = scene.add_entity(
+        morph=gs.morphs.Box(
+            size=(1.0, 1.0, 1.0),  # volume = 1 m^3
+            pos=(0.0, 0.0, 1.1),
+        ),
+        material=gs.materials.Rigid(rho=1.0),  # mass = 1 kg
+    )
+
+    bool_sensor = scene.add_sensor(gs.sensors.Contact(entity_idx=box.idx))
+    force_sensor = scene.add_sensor(
+        gs.sensors.ContactForce(
+            entity_idx=box.idx,
+            min_force=0.01,
+            max_force=(10.0, 20.0, -GRAVITY / 2),
+            noise=NOISE,
+            bias=BIAS,
+            random_walk=(NOISE * 0.01, NOISE * 0.02, NOISE * 0.03),
+            delay=0.05,
+            jitter=0.01,
+            interpolate=True,
+        )
+    )
+
+    scene.build()
+
+    scene.step()
+
+    assert not bool_sensor.read().any(), "RigidContactSensor should not be in contact with the ground yet."
+    assert_array_equal(
+        force_sensor.read_ground_truth(),
+        0.0,
+        err_msg="RigidContactForceSensor ground truth should be zero before contact.",
+    )
+    assert_allclose(
+        force_sensor.read(),
+        BIAS,
+        tol=NOISE * 2,
+        err_msg="RigidContactForceSensor should only read bias and small amount of noise before contact.",
+    )
+
+    for _ in range(100):
+        scene.step()
+
+    assert bool_sensor.read().all(), "Sensor should detect contact with the ground"
+    assert_allclose(
+        force_sensor.read_ground_truth(),
+        torch.tensor([0.0, 0.0, -GRAVITY]),
+        tol=1e-6,
```

Comment:
Why not use `tol` (the fixture parameter that you can pass as input argument of your test) ? You should document this if 1e-6 is necessary for the test to pass and try to explain why.